src/frontend/src/instance.rs

```diff
@@ -289,13 +288,10 @@ impl Instance {
         requests: InsertRequests,
         ctx: QueryContextRef,
     ) -> Result<Output> {
-        let _ = future::join_all(
-            requests
-                .inserts
-                .iter()
-                .map(|x| self.create_or_alter_table_on_demand(ctx.clone(), x)),
-        )
-        .await;
+        for req in requests.inserts.iter() {
```

Comment:
why change it to serialized execution

Reply:
On receiving a large prometheus request for the first time, creating numerous tables at once can cause huge pressure on etcd, slowing or even taking down the meta cluster, which then causing the whole greptimedb cluster unresponsive.

Changing it back to serialized can ease the pressure. A better way would to create a queue for table creation(or maybe control the queue on meta-srv after create table procedure is in use)